openapi/notion/client.bal

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/url;
+import ballerina/lang.'string;
+
+# Client configuration for Notion Connector
+#
+# + authConfig - Bearer token configuration
+# + secureSocketConfig - SSL configuration 
+public type ClientConfig record {
+    http:BearerTokenConfig authConfig;
+    http:ClientSecureSocket secureSocketConfig?;
+};
+
+# Ballerina Client for Notion API
+#
+# + clientEp - Connector http endpoint
+public client class Client {
+    http:Client clientEp;
+    # Client initialization.
+    #
+    # + clientConfig - Client configuration details
+    # + serviceUrl - Connector server URL
+    # + return - Returns error at failure of client initialization
```

Comment:
No need to say `Returns`. Shall we update code generation template?

Reply:
This was added by the Open API tool